subfun/make4Dmaps.m

```diff
@@ -0,0 +1,141 @@
+function make4Dmaps(degreeOfSmoothing,opt,isMVPA,deleteIndBeta,deleteIndTmaps,deleteResMaps)
```

Comment:
Not sure what one is for.

But I will trust you that it is necessary.

Reply:
the SPM creates individual t-maps for each regressor (or each condition in each run) which results in a large number of individual t-maps. 
This will merge all the t-maps of interest coming from the individual conditions and create 1 4D map that will be used later in the MVPA analysis. 
we change change the location of this file later, and (or better) have a seperate folder for different kinds of analyses as PPI, MVPA, beta estimation, etc. 
So we can have some modular folders as well for easiness.